js/router.js

```diff
@@ -48,6 +48,10 @@ module.exports = Backbone.Router.extend({
         this.navigate(translatedRoute, { trigger: true });
       });
     });
+    
+    history.size = -1;
```

Comment:
Access to history size and position from every file may be usefull in the future.